openpype/hosts/maya/api/lib_renderproducts.py

```diff
@@ -196,12 +196,18 @@ def __init__(self, layer, render_instance):
         """Constructor."""
         self.layer = layer
         self.render_instance = render_instance
-        self.multipart = False
+        self.multipart = self.get_multipart()
 
         # Initialize
         self.layer_data = self._get_layer_data()
         self.layer_data.products = self.get_render_products()
 
+    def get_multipart(self):
+        raise NotImplementedError(
```

Comment:
After some more though, maybe we could do:
```py
self.log.warn(
   "The render product implementation does not have a "
   "\"get_multipart\" method."
)
return False
```

that way we don't need to do https://github.com/ynput/OpenPype/pull/4497/files#diff-3b0eed55b2558684bcb1a84202dc932177842e1443bf2228020e1120305c57f0R1179